pkgs/desktops/gnome-3/extensions/draw-on-your-screen/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, fetchgit, gettext, gnome3 }:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-draw-on-your-screen";
+  version = "5";
+
+  src = fetchgit {
+    url = "https://framagit.org/abakkk/DrawOnYourScreen/";
+    rev = "v${version}";
+    sha256 = "0z4h2inwbwil68qhg9yy18mbi5nbf0ai8xdpvjy1xkaiwfhi4vmy";
+  };
+
+  uuid = "drawOnYourScreen@abakkk.framagit.org";
+
+  nativeBuildInputs = [
+    gettext
+  ];
+
+  installPhase = ''
+    mkdir -p $out/share/gnome-shell/extensions/${uuid}
+    cp -r . $out/share/gnome-shell/extensions/${uuid}
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A drawing extension for GNOME Shell";
+    license = licenses.gpl2;
```

https://framagit.org/abakkk/DrawOnYourScreen/-/blob/17c0da57a16619536f84e4879c23973ca41ab9c9/draw.js#L10-13

```suggestion
    license = licenses.gpl2Plus;
```